### ISSUE TITLE
Job file and render output location is incorrect - worker unable to render

### ISSUE THREAD

**Issue by bigsmalloverall** [dblanque/flamenco-compositor-script#2]
Job with multilayer export is successfully queued in Flamenco manager, but fails to render since workers are unable to locate files.

I'm unable to resolve this issue. I assume I did something wrong in config file. Please advise.

Blender version: 4.1.1, branch: blender-v4.1-release, commit date: 2024-04-15 15:11, hash: e1743a0317bc, type: Release build date: 2024-04-15, 23:33:30
System: 'Windows-10-10.0.22631-SP0'
Flamenco version: 3.4

Worker output log:
```bash
2024-05-05T15:12:50+02:00 Task assigned to worker DESKTOP-ABCD ()
2024-05-05T15:12:50+02:00 task changed status queued -> active
going to run:

    blender -b -y -P '{clientStoragePath}/jobs/startup_script.py' '{clientStoragePath}/jobs/2024-05-05-150314.328670-Test_multipass/untitled.flamenco.blend' -noaudio --render-output '{clientStoragePath}/render/2024-05-05-150314.328670-Test_multipass/######' --render-format OPEN_EXR_MULTILAYER --render-frame 1..5 --python-expr 'import bpy; bcs = bpy.context.scene; bcs.render.use_compositing = True; bcs.use_nodes = True' -- --custom-script --device-type OPTIX

pid=3848 > Blender 4.1.1 (hash e1743a0317bc built 2024-04-15 23:33:30)
pid=3848 > Read prefs: "C:\Users\User\AppData\Roaming\Blender Foundation\Blender\4.1\config\userpref.blend"
pid=3848 > OSError: Python file "Z:\Flamenco\Worker\{clientStoragePath}\jobs\startup_script.py" could not be opened: No such file or directory
pid=3848 > Error: Cannot read file "Z:\Flamenco\Worker\{clientStoragePath}\jobs\2024-05-05-150314.328670-Test_multipass\untitled.flamenco.blend": No such file or directory
pid=3848 > 
pid=3848 > Blender quit
Failed: command exited abnormally with code 1
2024-05-05T15:12:51+02:00 Task failed by worker DESKTOP-ABCD (), Manager will fail the entire job as there are no more workers left for tasks of type "blender".
2024-05-05T15:12:51+02:00 task changed status active -> failed
```

Here is how my shared storage is set up:
![Files](https://github.com/dblanque/flamenco-compositor-script/assets/87319288/9b753a62-4dd8-41a5-a6d6-ff746c16f58c)

* Files - folder where blend files are sent
* Renders - folders where renders are stored
* Worker - where all flamenco files reside

![image](https://github.com/dblanque/flamenco-compositor-script/assets/87319288/d22e7937-18da-48aa-875f-74d8b10cf672)

![Worker folder](https://github.com/dblanque/flamenco-compositor-script/assets/87319288/47277f35-c31c-4d20-ba33-0d8ca6739ce1)
The python file is inside `Worker` folder. The `multi_pass_render.js` in `scripts` folder.
![Scripts folder](https://github.com/dblanque/flamenco-compositor-script/assets/87319288/162765b2-431f-4ef6-9a3c-6de1c3d04f27)

My blender setup:
![image](https://github.com/dblanque/flamenco-compositor-script/assets/87319288/2635457e-06cc-43d8-bafc-91e9e295d85a)
![image](https://github.com/dblanque/flamenco-compositor-script/assets/87319288/19cab3e4-972f-44d2-9f1b-528a03951dc9)

flamenco-manager.yaml:
```yaml
# Configuration file for Flamenco.
# For an explanation of the fields, refer to flamenco-manager-example.yaml
#
# NOTE: this file will be overwritten by Flamenco Manager's web-based configuration system.
#
# This file was written on 2024-04-12 16:32:29 +02:00 by Flamenco 3.4

_meta:
  version: 3
manager_name: Flamenco
database: flamenco-manager.sqlite
database_check_period: 1h0m0s
listen: :12055
autodiscoverable: true
local_manager_storage_path: ./flamenco-manager-storage
shared_storage_path: Z:\Flamenco\Files
shaman:
  enabled: false
  garbageCollect:
    period: 24h0m0s
    maxAge: 744h0m0s
    extraCheckoutPaths: []
task_timeout: 10m0s
worker_timeout: 1m0s
blocklist_threshold: 3
task_fail_after_softfail_count: 3
variables:
  blender:
    values:
    - platform: linux
      value: blender
    - platform: windows
      value: blender
    - platform: darwin
      value: blender
  blenderArgs:
    values:
    - platform: all
      value: -b -y
  storagePath:
    values:
    - platform: windows
      value: "Z:\\Flamenco\\Files"
  jobSubPath:
    values:
    - platform: all
      value: jobs
  renderSubPath:
    values:
    - platform: all
      value: render
  deviceType:
    values:
    - platform: all
      value: "OPTIX"
```

**Comment by bigsmalloverall** [comment]
Found the problem. Provided [flamenco-manager-example.yaml](https://github.com/dblanque/flamenco-compositor-script/blob/main/flamenco-manager-example.yaml) is invalid. The variable `storagePath` should be `clientStoragePath`.

**Comment by dblanque** [comment]
Fixed in PR #3 (by you!).

**Comment by dblanque** [comment]
I went ahead and copied the one from the actual README, since that's the one that was the most up to date.
Let me know if there are any more issues, and thank you for the contribs.
